packages/clay-popover/src/index.tsx

```diff
@@ -15,15 +17,41 @@ export const ALIGN_POSITIONS = [
 	'bottom-left',
 	'bottom-right',
 	'left',
+	'left-top',
+	'left-bottom',
 	'right',
+	'right-top',
+	'right-bottom',
 ] as const;
 
+const ALIGNMENTS_MAP = {
+	bottom: ['tc', 'bc'],
+	'bottom-left': ['tl', 'bl'],
+	'bottom-right': ['tr', 'br'],
+	left: ['cr', 'cl'],
```

Comment:
How come we mix and match the nomenclature of these attributes? Some have quotes and others don't. I understand that the ones that don't have them don't need them, but why not favor consistency? 

> Yeah, but your scientists were so preoccupied with whether or not they could, they didn't stop to think if they should.

Reply:
I guess the idea is "use it when it's needed don't when it's not".

Reply:
Yeah I think we would tend to follow juliens comment. Also we have an eslint rule in this project(`quote-props`) that auto formats it to not use quotes when it isn't necessary.

Here is an example eslint error
<img width="641" alt="Screen Shot 2020-01-10 at 11 25 34 AM" src="https://user-images.githubusercontent.com/6843530/72180525-35dfde00-339c-11ea-90ba-f6525e818431.png">

Reply:
The only reason we haven't changed it to "consistent-as-needed" yet:

https://github.com/liferay/eslint-config-liferay/pull/39

is due to an open bug in Prettier:

https://github.com/prettier/prettier/issues/6064

Once that is resolved this will become somewhat nicer:

```js
const ALIGNMENTS_MAP = {
	'bottom': ['tc', 'bc'],
	'bottom-left': ['tl', 'bl'],
	'bottom-right': ['tr', 'br'],
	'left': ['cr', 'cl'],
        // etc...
};